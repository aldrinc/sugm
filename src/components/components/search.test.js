import React from 'react';
import Search from './search';
import renderer from 'react-test-renderer';

it('search renders correctly', () => {
    const tree = renderer.create(
      <Search></Search>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
