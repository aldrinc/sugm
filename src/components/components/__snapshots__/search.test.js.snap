// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search renders correctly 1`] = `
<form>
  <input
    onChange={[Function]}
    onClick={[Function]}
    placeholder="What can we help you find?"
    type="text"
  />
</form>
`;
